server/tests/employer.site.test.js

```diff
@@ -0,0 +1,227 @@
+const { ValidationError } = require('yup');
+const { v4 } = require('uuid');
+const app = require('../server');
+const {
+  saveSingleSite, saveSites, getSites, getSiteByID, updateSite,
+} = require('../services/employers.js');
+const {
+  getParticipants,
+  makeParticipant,
+  setParticipantStatus,
+  getHiredParticipantsBySite,
+} = require('../services/participants.js');
+
+const { startDB, closeDB } = require('./util/db');
+
+describe('Employer Site Endpoints', () => {
+  let server;
+
+  beforeAll(async () => {
+    await startDB();
+    server = app.listen();
+  });
+
+  afterAll(async () => {
+    await closeDB();
+    server.close();
+  });
+
+  const employerAId = v4();
+  const employerBId = v4();
+  const participant1 = {
+    maximusId: 648690,
+    lastName: 'Extra',
+    firstName: 'Eddy',
+    postalCode: 'V1V2V3',
+    postalCodeFsa: 'V1V',
+    phoneNumber: '2502223333',
+    emailAddress: 'eddy@example.com',
+    interested: 'yes',
+    nonHCAP: 'yes',
+    crcClear: 'yes',
+    preferredLocation: 'Fraser',
+  };
+
+  const participant2 = {
+    maximusId: 648691,
+    lastName: 'Finkleman',
+    firstName: 'Freduardo',
+    postalCode: 'V1V2V3',
+    postalCodeFsa: 'V1V',
+    phoneNumber: '2502223333',
+    emailAddress: 'freddy@example.com',
+    interested: 'yes',
+    nonHCAP: 'yes',
+    crcClear: 'yes',
+    preferredLocation: 'Fraser',
+  };
+
+  const site = {
+    siteId: 67,
+    siteName: 'Test site',
+    phaseOneAllocation: 1,
+    address: '123 XYZ',
+    city: 'Victoria',
+    healthAuthority: 'Vancouver Island',
+    postalCode: 'V8V 1M5',
+    registeredBusinessName: 'AAA',
+    operatorName: 'Test Operator',
+    operatorContactFirstName: 'AABB',
+    operatorContactLastName: 'CCC',
+    operatorEmail: 'test@hcpa.fresh',
+    operatorPhone: '2219909090',
+    siteContactFirstName: 'NNN',
+    siteContactLastName: 'PCP',
+    siteContactPhoneNumber: '2219909091',
+    siteContactEmailAddress: 'test.site@hcpa.fresh',
+  };
+
+  it('Create new single site, receive success', async () => {
```

Comment:
This test and the one below are new, everything else is copied from employer form tests (should be separate commits for easy review)